descriptors/ModuleDescriptor-template.json

```diff
@@ -62,7 +66,8 @@
             "marc-migrations.operations.item.put"
           ],
           "modulePermissions": [
-            "inventory-storage.authorities.bulk.post"
+            "inventory-storage.authorities.bulk.post",
+            "inventory-storage.instances.bulk.post"
```

Comment:
I didn't find an endpoint with this permission. Does it exist? If no, I think having this permission here could break platform build as the module is referencing to not existing permission.

Reply:
You are right, this permission will be added in scope of https://folio-org.atlassian.net/browse/MODINVSTOR-1225 . Thank you